util/topgen/lib.py

```diff
@@ -21,24 +21,38 @@
 from .intermodule import get_dangling_im_def # noqa : F401 # isort:skip
 
 
-class Name(object):
-    """We often need to format names in specific ways; this class does so."""
+class Name():
```

Comment:
`Name()` -> `Name` (no need for parentheses)

Reply:
jep, removed